frameworks/mastra/supervisor-agent/README.md

```
@@ -0,0 +1,129 @@
+<div align="center">
+    <img src="https://raw.githubusercontent.com/agentuity/cli/refs/heads/main/.github/Agentuity.png" alt="Agentuity" width="100"/> <br/>
+    <strong>Build Agents, Not Infrastructure</strong> <br/>
+    <br/>
+        <a target="_blank" href="https://app.agentuity.com/deploy" alt="Agentuity">
+            <img src="https://app.agentuity.com/img/deploy.svg" /> 
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)